glusterd2/volgen2/store-utils.go

```diff
@@ -29,8 +30,22 @@ func GetVolfiles() ([]string, error) {
 	volfiles := make([]string, len(resp.Kvs))
 
 	for i, kv := range resp.Kvs {
-		volfiles[i] = string(kv.Key)
+		volFile := strings.TrimPrefix(string(kv.Key), "volfiles/")
+		volfiles[i] = volFile
 	}
 
 	return volfiles, nil
 }
+
+//GetVolfile return perticular volfile info
```

Comment:
type: particular

The repo has [Go report card](https://goreportcard.com/report/github.com/gluster/glusterd2) enabled and one of the tests there is misspell test. If this is merged, this will be caught there. As a bonus, when you refresh this PR, you can also fix other misspells that Go report card reported.